framework/math/parallel_vector/parallel_stl_vector.cc

```diff
@@ -81,7 +77,7 @@ ParallelSTLVector::LocalSTLData()
 std::vector<double>
 ParallelSTLVector::MakeLocalVector()
 {
-  return std::vector<double>(values_.begin(), values_.begin() + scint(local_size_));
+  return std::vector<double>(values_.begin(), values_.begin() + static_cast<int>(local_size_));
```

Comment:
For my own edification... Are static casts necessary in scenarios like this? I see them somewhat inconsistently sprinkled throughout the code and I'm never sure whether I should use them or not. This is another standardization thing to tackle in the future.

Reply:
`clang-tidy` complains about narrowing conversions, so I think that's where these casts are coming from. In CLion, you should be able to see those "warning" in the [Problems tool](https://www.jetbrains.com/help/clion/problems-tool-window.html)